diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter2/comments/CommentedCodeTest.kt

```diff
@@ -256,4 +256,42 @@ class CommentedCodeTest : LintTestBase(::CommentsRule) {
             """.trimMargin(),
             LintError(1, 1, ruleId, "${COMMENTED_OUT_CODE.warnText()} public fun someFunc(name: String): Boolean =", false))
     }
+
+
+    @Test
+    @Tag(WarningNames.COMMENTED_OUT_CODE)
+    fun `should not trigger on multiline comments #1`() {
+        lintMethod(
+                """
+            |/*
```

Comment:
```suggestion
            |   /*
```
Wasn't the bug in case leading or trailing spaces are present? Or did it trigger on simple comments too?

Reply:
It didn't trigger on simple comment, neither it triggers if we add spaces in this test. When I was testing with fix tests and different yml, I found out that it adds leading spaces to `BLOCK_COMMENT`.

Reply:
You should reproduce it in auto tests too, in smoke test, if I understand correctly. There is currently an option to disable rules by editing config in smoke test, you can enhance it to allow custom configuration of rules.

Reply:
Done